libs/flowbite-angular/dropdown/dropdown.theme.ts

```diff
@@ -72,10 +72,10 @@ export const dropdownTheme: DropdownTheme = createTheme({
       disabled: 'hidden',
     },
     placement: {
-      'top-center': '',
-      'bottom-center': '',
-      'left-center': '',
-      'right-center': '',
+      'top-center': 'bottom-full mb-2.5 left-1/2 -translate-x-1/2',
```

Comment:
I forgot during the refacto to mention that sufixes are going to be removed
So instead of top-center it's just top (and so on for the other)
You have to update the position definition

Reply:
![kép](https://github.com/user-attachments/assets/f4d5ee28-017c-4f18-9b83-9487e6127215)
Did i messed up something with the rebase, or is it not on main yet?

Reply:
No it's not you
I messed on my commits I've done them on main instead of 1.1.0...
I'll check if something break or what don't worry :)